gcip/core/pipeline.py

```diff
@@ -76,6 +76,13 @@ def render(self) -> Dict[str, Any]:
 
         pipeline["stages"] = list(stages.keys())
         for job in job_copies:
+            if job.name in pipeline:
+                raise ValueError(
```

Comment:
Is this really a ValueError?
Maybe it would be better to rase a own exception like `JobNameError`.
That would help to provide a better handling.

Reply:
solved with bb569c5